src/Fixer/PhpUnit/PhpUnitAttributesFixer.php

```diff
@@ -87,6 +104,16 @@ public function getPriority(): int
         return 8;
     }
 
+    protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('keep_annotations', 'To keep annotations or not.'))
```

Comment:
```suggestion
            (new FixerOptionBuilder('keep_annotations', 'Whether to keep annotations or not. It may be helpful in projects that have to support variety of PHP versions, including pre-8.0.'))
```

Project-wide we have "Whether ..." convention. Also in this case I think it's a good idea to provide reasoning behind the flag.

Reply:
I would tell here it is helpful/needed for keeping support for PHPUnit 9 or lower.